handbook/support/p4-enablement.md

```diff
@@ -0,0 +1,163 @@
+# Dogfood Perforce Server
+
+Perforce is a version control system like git, subversion, or mercurial. While git is based on a distributed, decentralised model, Perforce is centralised. For testing purposes you may use our perforce dogfood server.
+
+## Setting Up
+- To connect to perforce you'll need the perforce cli installed locally. Use the command: `brew install --cask perforce`
+- The following environment variables configure your shell to point at the perforce server. Set them to your `env` with the `export` command, or add them to your `.bashrc` or `.zshrc` file.
+```
+# .zshrc or .bashrc
+
+P4PORT=perforce.sgdev.org:1666 # This points the p4 cli at the dogfood server
+P4USER=admin # sets your user
+P4PASSWD=<login session ticket value explained below> # doesn't require string quotes
+P4EDITOR=/usr/bin/vim # specifies the editor opened by some p4 commands
+```
+## Interacting With Perforce Dogfood
+
+To add repos to the perforce dogfood server follow the following procedure:
+- [Dogfood Perforce Server](#dogfood-perforce-server)
+  - [Setting Up](#setting-up)
+  - [Interacting With Perforce Dogfood](#interacting-with-perforce-dogfood)
+    - [Generate A Session Ticket](#generate-a-session-ticket)
+    - [Create New Depot](#create-new-depot)
+    - [Create A Client](#create-a-client)
+    - [Add Files and Submit](#add-files-and-submit)
+    - [Configuring Sourcegraph To Sync Dogfood Depots](#configuring-sourcegraph-to-sync-dogfood-depots)
+- [Local Perforce Server](#local-perforce-server)
+
+### Generate A Session Ticket
```

Comment:
We use sentence case in headings (not going to suggest changes to remaining of this doc 😁 )

Reply:
Thanks for giving it a thorough look over 🙏